homeassistant/components/samsungtv/bridge.py

```diff
@@ -223,9 +224,13 @@ def try_connect(self):
                 return RESULT_SUCCESS
             except WebSocketException:
                 LOGGER.debug("Working but unsupported config: %s", config)
```

Comment:
Minor, but I would suggest dropping a more meaningful message here, something like:
LOGGER.debug("Timeout hit while trying the following config: %s", config)

Reply:
Yes maybe it's a bit confusing but it describes exactly the problem: the connection is working but something went wrong, and if already the login fails it's apparently not like we support it.